js/bleutrade.js

```diff
@@ -203,4 +212,108 @@ module.exports = class bleutrade extends bittrex {
             throw new ExchangeError (this.id + ' publicGetOrderbook() returneded no result ' + this.json (response));
         return this.parseOrderBook (orderbook, undefined, 'buy', 'sell', 'Rate', 'Quantity');
     }
+
+    parseTrade (trade, market = undefined) {
+        let parsedTrade = super.parseTrade (trade, market);
+        parsedTrade.id = trade['ID'];
```

Comment:
This is not needed anymore, because the following has been added already: https://github.com/ccxt/ccxt/pull/4154/files#diff-ca765a07d49756ae1107fcb6fb004461R403 ;)

Reply:
Thanks.  I removed it.